config/locales/en.yml

```diff
@@ -25,3 +25,178 @@ en:
       %{logo_img_tag}
       <p class="lead">OnDemand provides an integrated, single access point for all of your HPC resources.</p>
     motd_title: "Message of the Day"
+    breadcrumbs:
+      home: "Home"
+      my_sessions: "My Interactive Sessions"
+      all_apps: "All Apps"
+    restart_msg_tag: |
+      Your group membership has changed, which affects your access to apps. Your dashboard should restart automatically. 
+      If it doesn't please, click: <a href="%{restart_url}">restart web server</a>.
+    nav:
+      sessions: "My Interactive Sessions"
+      all_apps: "All Apps"
+      develop:
+        title: "Develop"
+        docs: "Developer Documentation"
+      help:
+        title: "Help"
+        change_password: "Change HPC Password"
+        support: "Contact Support"
+        docs: "Online Documentation"
+        two_factor: "Configure Two-Factor Authentication"
+      restart_server: "Restart Web Server"
+      user: "Logged in as %{username}"
+      logout: "Log Out"
+    bc:
+      no_sessions: "You have no active sessions."
+      no_apps: "There are no available Interactive Apps."
+      sandbox: " [Sandbox]"
+      sessions:
+        data: |
+          * The %{title} session data for this session can be accessed 
+          under the %{data_link_tag}.
+        staged_root: "staged root directory"
+        status:
+          bad: |
+            Your session has entered a bad state. Feel free to contact support for 
+            further information.
+          missing_connection: |
+            This app is missing information required to establish a connection. Please 
+            contact support if you see this message.
+          queued: |
+            Please be patient as your job currently sits in queue. The wait time depends 
+            on the number of cores as well as time requested.
+          starting: |
+            Your session is currently starting... Please be patient as this process can 
+            take a few minutes.
+        native:
```

Comment:
I would actually remove this native section in a separate isolated commit - along with the references in the views of course, so that we can revert the change later. The reason is that native vnc for Open OnDemand is not a documented or supported feature yet, and if admins are using this YAML as a reference for what text they can change, this makes up a large portion of the page for an undocumented feature.

It is important to revert just this and the corresponding views in an isolated commit, so that in the future we will have the ability to easily revert undoing that change if we do decide to support native vnc.